compiler-core/src/parse/error.rs

```diff
@@ -411,6 +421,10 @@ pub enum ParseErrorType {
     TypeConstructorNoArguments,           // let a : Int()
     TypeDefinitionNoArguments,            // pub type Wibble() { ... }
     UnknownAttributeRecordVariant, // an attribute was used that is not know for a custom type variant
+    PythonicImport {
+        mod_or_namespace: EcoString,
+        importee: EcoString,
```

Comment:
What's an importee?

Document this error please 🙏

Reply:
This is me probably abusing the English language since `ee` usually indicates the subject of the verb-stem. So in this case, it's the item being imported.

Given the recommendation to rename `mod_or_namespace` to `module`, I think it's likely clearer to rename `importee` to `item`